tutorials/video_classification_example/finetune.py

```diff
@@ -0,0 +1,255 @@
+from pathlib import Path
+from argparse import Namespace
+from torchvision.transforms._transforms_video import CenterCropVideo
+from pytorchvideo.data import LabeledVideoDataset
+from pytorchvideo.data.clip_sampling import UniformClipSampler
+import pytorch_lightning as pl
+import torch
+from pytorchvideo.models.head import create_res_basic_head
+from torch import nn
+from torch.optim import Adam
+
+# HACK
+from train import *
+
+
+class UCF11DataModule(KineticsDataModule):
+
+    def __init__(
+        self,
+        root="./",
+        batch_size=32,
+        num_workers=8,
+        holdout_scene=None,
+        side_size = 256,
+        crop_size = 256,
+        clip_mean = (0.45, 0.45, 0.45),
+        clip_std = (0.225, 0.225, 0.225),
+        num_frames = 8,
+        sampling_rate = 8,
+        frames_per_second = 30
+    ):
+        super().__init__(Namespace(data_type='video', batch_size=batch_size, workers=num_workers))
+
+        self.root = Path(root) / 'action_youtube_naudio'
+        assert self.root.exists(), "Dataset not found."
+        self.batch_size = batch_size
+        self.num_workers = num_workers
+        self.holdout_scene = holdout_scene
+        self.side_size = side_size
+        self.mean = clip_mean
+        self.std = clip_std
+        self.crop_size = crop_size
+        self.num_frames = num_frames
+        self.sampling_rate = sampling_rate
+        self.frames_per_second = frames_per_second
+        self.clip_duration = (self.num_frames * self.sampling_rate) / self.frames_per_second
+
+        self.classes = [x.name for x in self.root.glob("*") if x.is_dir()]
+        self.id_to_label = dict(zip(range(len(self.classes)), self.classes))
+        self.class_to_label = dict(zip(self.classes, range(len(self.classes))))
+        self.num_classes = len(self.classes)
+
+
+        # TODO - too many repeated .glob calls here.
+        self.train_paths = []
+        self.val_paths = []
+        self.holdout_scenes = {}
+        for c in self.classes:
+
+            # Scenes within each class directory
+            scene_names = sorted(set(x.name for x in (self.root / c).glob("*") if x.is_dir() and x.name != 'Annotation'))
```

Comment:
I'm fairly confused about some of these lines, I think it's because i'm not clear on the directory structure you're reading from. What is? E.g. What is the "Annotation" filter

Reply:
Yea this line is definitely messed up haha. Fixed in newer version.

The root dir has directories with different class names. within the class directories there are scene directories that separate out clips shot by the same actor/in the same scene. The `Annotation` dir is not a valid scene, so I'm ignoring it when gathering their names.

![ucf11-structure](https://user-images.githubusercontent.com/32437151/118207121-2efa2d00-b421-11eb-8aaa-dbbf0aa0d023.png)